azurerm/helpers/azure/app_service.go

```diff
@@ -451,5 +533,9 @@ func FlattenAppServiceSiteConfig(input *web.SiteConfig) []interface{} {
 	result["ftps_state"] = string(input.FtpsState)
 	result["min_tls_version"] = string(input.MinTLSVersion)
 
+	if input.Cors != nil {
+		result["cors"] = FlattenAppServiceCorsSettings(input.Cors)
+	}
```

Comment:
can we remove the if statement around this so that this is always set (we can return an empty `[]interface{}` from inside the flatten method)